packages/logger/src/writers/console.ts

```diff
@@ -0,0 +1,124 @@
+import { LogLevel } from '../logger';
+import { Attributes, logLevelToString, truthyEnv } from '../utils';
+import { jsonStringify, LogWriter } from './common';
+
+const asciMap = {
+  timestamp: '\x1b[90m', // bright black
+  trace: '\x1b[36m', // cyan
+  debug: '\x1b[90m', // bright black
+  info: '\x1b[32m', // green
+  warn: '\x1b[33m', // yellow
+  error: '\x1b[41;39m', // red; white
+  message: '\x1b[1m', // bold
+  key: '\x1b[35m', // magenta
+  reset: '\x1b[0m', // reset
+};
+
+export interface ConsoleLogWriterOptions {
+  /** @default globalThis.Console */
+  console?: Pick<Console, 'debug' | 'info' | 'warn' | 'error'>;
+  /**
+   * Whether to disable colors in the console output.
+   *
+   * @default env.NO_COLOR || false
+   */
+  noColor?: boolean;
+  /**
+   * Whether to include the timestamp at the beginning of the log message.
+   *
+   * @default false
+   */
+  noTimestamp?: boolean;
+}
+
+export class ConsoleLogWriter implements LogWriter {
```

Comment:
Do we still respect custom inspection `Symbol.for('nodejs.util.inspect.custom');` or from now on we just use `JSON.stringify`?

Reply:
Nvm found it! Great!

Reply:
[We do](https://github.com/graphql-hive/gateway/blob/a39189a656671ed9ab571bd39edba5c94524410d/packages/logger/src/utils.ts#L131-L162), object inspection is now implemented in the logger itself and not the writers ([see `parseAttrs`](https://github.com/graphql-hive/gateway/blob/a39189a656671ed9ab571bd39edba5c94524410d/packages/logger/src/utils.ts#L57C17-L57C27)).